src/vstest.console/vstest.console.csproj

```diff
@@ -60,5 +60,10 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net451'">
+    <PackageReference Include="Microsoft.VisualStudio.Telemetry">
+      <Version>15.3.799-masterDDDBA9E4</Version>
```

Comment:
><Version>15.3.799-masterDDDBA9E4</Version> [](start = 6, length = 42)

Does not seem right. How did we get this version?

Reply:
This is the most downloaded version. So used it.